.drone.star

```diff
@@ -677,11 +673,8 @@ def e2eTests(ctx):
         depends_on = []
         steps = skipIfUnchanged(ctx, "e2e-tests") + \
                 restoreBuildArtifactCache(ctx, "pnpm", ".pnpm-store") + \
-                restoreBuildArtifactCache(ctx, "playwright", ".playwright") + \
                 installPnpm() + \
-                installPlaywright() + \
```

Comment:
don't need to install browser from playwright. the image used in test run step has the required chrome browser